.github/workflows/pytest-gpu.yml

```diff
@@ -78,7 +78,10 @@ jobs:
 
     - name: Build docker image
       run: |
-          docker build . --pull --file docker/Dockerfile.devito --tag devito_img --build-arg base=${{ matrix.base }}
+          docker build . --rm --pull --file docker/Dockerfile.devito --tag devito_img --build-arg base=${{ matrix.base }}
```

Comment:
Isn't `--rm` a run option

Reply:
it's a build too -- it removes the dangling layers!